### ISSUE TITLE
Timed-Text WG 2022

### ISSUE THREAD

**Issue by himorin** [w3c/strategy#290]
New charter proposal, reviewers please take note.

# Charter Review

[Charter:](https://w3c.github.io/charter-timed-text/)

What kind of charter is this? Check the relevant box / remove irrelevant branches. 

* New
  - [ ] New WG 
  - [ ] New IG
  - If this is a new WG or IG charter request, link to [Advance Notice](), and any issue discussion: 

* Existing 
  - [x] Existing WG recharter 
  - [ ] Existing IG recharter
  - If this is a charter extension or revision, link a [diff from previous charter](https://services.w3.org/htmldiff), and any issue discussion:

- current charter: [to 2021/Dec/31](https://www.w3.org/2020/12/timed-text-wg-charter.html)
- [diff](https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fwww.w3.org%2F2020%2F12%2Ftimed-text-wg-charter.html&doc2=https%3A%2F%2Fw3c.github.io%2Fcharter-timed-text%2F)

Horizontal Reviews: apply the Github label "Horizontal review requested" to request reviews for accessibility (a11y), internationalization (i18n), privacy, and security.  Also add a "card" for this issue to the [Strategy Funnel](https://github.com/w3c/strategy/projects/2).

Communities suggested for outreach: (no new)

Known or potential areas of concern: 

Where would charter proponents like to see issues raised? : [github](https://github.com/w3c/charter-timed-text)

Anything else we should think about as we review?

**Comment by himorin** [comment]
- section 2.1 would be removed
- one update is planned for success criteria section: https://github.com/w3c/charter-timed-text/pull/67/files

**Comment by samuelweiler** [comment]
I added a card for this issue in the pipeline.

> * section 2.1 would be removed
> * one update is planned for success criteria section: https://github.com/w3c/charter-timed-text/pull/67/files

@himorin, given the above and the generally rough shape of the charter, I encourage you to remove the horizontal-review-requested label for now and add it again when you think the charter is closer to ready.

**Comment by himorin** [comment]
> > * section 2.1 would be removed
> > * one update is planned for success criteria section: https://github.com/w3c/charter-timed-text/pull/67/files
> 
> @himorin, given the above and the generally rough shape of the charter, I encourage you to remove the horizontal-review-requested label for now and add it again when you think the charter is closer to ready.

@plehegar is it ok to remove the label for now? (should be short like within this week to get approved by chairs on changes...)

**Comment by samuelweiler** [comment]
Yes, it's okay.  :-)

**Comment by himorin** [comment]
No comment/request from i18n.

**Comment by plehegar** [comment]
Approved by W3M: https://www.w3.org/2022/02/16-w3m-minutes.html#t05

**Comment by svgeesus** [comment]
Existing charter was extended until 31 March 2023.
